### PR TITLE
Add metadata for CV for 23.10 and 23.4 CVE fixes

### PR DESCRIPTION
**Pull request by mihairaduvlad**

### CODE REVIEW

**Review by irinel-birsan-uipath — DISMISSED** [review]

File: metadata/ComputerVision__8__metadata.json [hunk 1]
```diff
@@ -19,8 +19,8 @@
   "tenantName": "UiPath",
   "minAIFabricVersion": "22.10",
   "languageVersion": 3,
-  "version": 5,
-  "customVersion": "23.04.4",
-  "imagePath": "cv2304:23.4.10.4",
+  "version": 8,
```
[the remark this answers is not in the document]
Comment (mihairaduvlad) [reply metadata/ComputerVision__8__metadata.json]: Done

File: metadata/ComputerVision__2304105__metadata.json [hunk 2]
```diff
@@ -19,8 +19,8 @@
   "tenantName": "UiPath",
   "minAIFabricVersion": "22.10",
   "languageVersion": 3,
-  "version": 7,
-  "customVersion": "23.10.1",
-  "imagePath": "cv2310:23.10.5.2",
+  "version": 2304105,
```
Comment (irinel-birsan-uipath) [inline metadata/ComputerVision__2304105__metadata.json:22]: This version is not displayed anywhere, why this change?

Comment (irinel-birsan-uipath) [reply metadata/ComputerVision__2304105__metadata.json:22]: When present, the custom version is displayed.

Comment (mihairaduvlad) [reply metadata/ComputerVision__2304105__metadata.json:22]: We want to change it to represent the CV Model version, so in the future it will be harder to put a number there presented in another branch in order to break something. Since this number is unique per cv release, it will also be unique. Does it sounds good?

Comment (irinel-birsan-uipath) [reply metadata/ComputerVision__2304105__metadata.json:22]: It sounds good but we need to check somehow that this is working.

I wouldn't do this change in stable.


Maybe next time, in testing, ane merge it in stable after testing.

Comment (mihairaduvlad) [reply metadata/ComputerVision__2304105__metadata.json]: Ok, I've reverted last changes

**Review by irinel-birsan-uipath — APPROVED** [review]

File: metadata/ComputerVision__2304105__metadata.json (hunk 2 above)
Comment (MaxTranced) [reply metadata/ComputerVision__2304105__metadata.json]: BTW, instead of `2304105` it should be either `23041005` or `230406`

Comment (irinel-birsan-uipath) [reply metadata/ComputerVision__2304105__metadata.json]: @MaxTranced maybe is working but we I would like to test in testing first before going to stable.

**Review by MaxTranced — APPROVED** [review]
lgtm, I've asked @mihairaduvlad to clarify this for the future
